json-rpc/docs/method_get_currencies.md

```diff
@@ -0,0 +1,53 @@
+## Method get_currencies
+
+**Description**
+
+Get information about various currencies supported by the Libra blockchain
+
+
+### Parameters
+
+None
+
+
+### Returns
+
+Returns array of [CurrencyInfo](#type-currencyinfo) objects.
+
+
+### Type CurrencyInfo
```

Comment:
do we want to move all types?  this would allow an single pane look on all the return objects, which maybe useful for sdk development.

Reply:
good idea, I split out major types, keep TransactionData type with Transaction type for now, trying to keep some relationship together. This one and metadata should be split out.